ls1intum/Artemis
# `Exam mode`: Center summary on wide screens

### Checklist
#### General


#### Client

### Motivation and Context



Closes #5026: Scale exam summary on 4k ultra wide screens.

### Description


In order to scale correctly, I centered the table.

### Steps for Testing

Prerequisites:
- 1 Student
- 1 Exam


1. Log in to Artemis
2. Participate as a student in the Exam
3. View the exam summary (should take the complete width)

### Review Progress





#### Code Review
#### Manual Tests

### Screenshots



![image](https://user-images.githubusercontent.com/60005702/168662937-9d4bda3c-017a-4681-82f8-43c7a257c21c.png)
